playbooks/roles/wordpress/README.md

```diff
@@ -0,0 +1,10 @@
+Ansible role for OpenCraft's landing page
+============================================================
+
+This role sets up a Wordpress-based landing page, maintenance page and default page. The service is run behind nginx, and
+certbot is used to manage SSL certificates for the landing page.
+
+After deploying new landing page, you need to:
+- mount volume with Wordpress data (it can be created from a snapshot),
+- create new MySQL DB from the dump and set it in `wp-config.php`,
```

Comment:
@Agrendalath, can this be automated if the path to the local database dump is provided?

Reply:
Sure, but we should do a follow-up for this as it significantly changes the current scope.